helper/resource/plugin.go

```diff
@@ -330,7 +329,7 @@ func runProviderCommand(ctx context.Context, t testing.T, f func() error, wd *pl
 	// started.
 	err := f()
 	if err != nil {
-		log.Printf("[WARN] Got error running Terraform: %s", err)
+		logging.HelperResourceWarn(ctx, "Error running Terraform CLI command", map[string]interface{}{logging.KeyError: err})
```

Comment:
Nothing to change here, but for when we go 1.18 here, we could consider using `map[string]any{...}` instead. It's still not ideal (I wish we could have variadic argument list that could enforce even-ness), but a bit shorter.

What do you think?

Again, there is nothing to change here.

Reply:
I do agree that once we're minimum Go 1.18 we should change all the `interface{}` usage to `any` 😉  https://github.com/hashicorp/terraform-plugin-sdk/issues/834

Reply:
Of course there is already a story for this :D